third-party/jemalloc/README

```diff
@@ -0,0 +1,11 @@
+==========================
+jemalloc README for Chapel
+==========================
+
+This copy of jemalloc is being released with Chapel for convenience and
+uses a snapshot from:
```

Comment:
Any particular reason? I don't think we do that for any other third-party packages